float64 timeStamp      # ROS time-stamp
float64[] forces        # Calibrated force values
